onchain/src/social/request.cairo

```diff
@@ -117,7 +89,7 @@ mod tests {
     }
 
     #[test]
-    fn test_wip3() {
+    fn verify_event2() {
```

Comment:
Make content = `nprofile1qys8wumn8ghj7un9d3shjtn2daukymme9e3k7mtdw4hxjare9e3k7mgqyzzxqw6wxqyyqqmv4rxgz2l0ej8zgrqfkuupycuatnwcannad6ayqx7zdcy send 1 USDC to nprofile1qqs2sa3zk4a49umxg4lgvlsaenrqaf33ejkffd78f2cgy4xy38h393s2w22mm`